app/Providers/BroadcastServiceProvider.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Providers;
+
+use Illuminate\Support\Facades\Broadcast;
+use Illuminate\Support\ServiceProvider;
+
+class BroadcastServiceProvider extends ServiceProvider
+{
+    /**
+     * Bootstrap any application services.
+     */
+    public function boot(): void
+    {
+        Broadcast::routes();
+
+        require base_path("routes/channels.php");
```

Comment:
Is app running fine, while this file doesn't exist?
Maybe we should remove this provider or add empty file in routes directory or just remove this line?

Reply:
removed